<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinVideoDisplayForm</class>
 <widget class="QWidget" name="LinVideoDisplayForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linguine Video</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="horizontalPage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="hVideoWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="hPlayButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="linresource.qrc">
             <normaloff>:/icons/playback_play_icon&amp;48.png</normaloff>:/icons/playback_play_icon&amp;48.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hStopButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="linresource.qrc">
             <normaloff>:/icons/playback_stop_icon&amp;48.png</normaloff>:/icons/playback_stop_icon&amp;48.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hInfoButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="linresource.qrc">
             <normaloff>:/icons/info_icon&amp;48.png</normaloff>:/icons/info_icon&amp;48.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="hPercentageLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="hTitleLabel">
         <property name="text">
          <string>Media Name Goes Here</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>GStreamer does not support portrait-mode video.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="vTitleLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Video Title</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="vProgressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="vPlayButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="linresource.qrc">
             <normaloff>:/icons/playback_pause_icon&amp;48.png</normaloff>:/icons/playback_pause_icon&amp;48.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="vStopButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="linresource.qrc">
             <normaloff>:/icons/playback_stop_icon&amp;48.png</normaloff>:/icons/playback_stop_icon&amp;48.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="vInfoButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="linresource.qrc">
             <normaloff>:/icons/info_icon&amp;48.png</normaloff>:/icons/info_icon&amp;48.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="linresource.qrc"/>
 </resources>
 <connections/>
</ui>
